docs/core/tools/dotnet-new-uninstall.md

```diff
@@ -9,58 +9,73 @@ ms.date: 04/29/2021
 
 ## Name
 
-`dotnet new --uninstall` - uninstalls a template package.
+`dotnet new uninstall` - uninstalls a template package.
 
 ## Synopsis
 
 ```dotnetcli
-dotnet new --uninstall <PATH|NUGET_ID>
+dotnet new uninstall <PATH|NUGET_ID> 
+    [-d|--diagnostics] [--verbosity <LEVEL>] [-h|--help]
 ```
 
 ## Description
 
-The `dotnet new --uninstall` command uninstalls a template package at the `PATH` or `NUGET_ID` provided. When the `<PATH|NUGET_ID>` value isn't specified, all currently installed template packages and their associated templates are displayed. When specifying `NUGET_ID`, don't include the version number.
+The `dotnet new uninstall` command uninstalls a template package at the `PATH` or `NUGET_ID` provided. When the `<PATH|NUGET_ID>` value isn't specified, all currently installed template packages and their associated templates are displayed. When specifying `NUGET_ID`, don't include the version number.
 
 > [!NOTE]
 > [!INCLUDE [new syntax](../../../includes/dotnet-new-7-0-syntax.md)]
 >
-> Examples of the new syntax:
->
-> - Show help for the `uninstall` subcommand.
->
->   ```dotnetcli
->   dotnet new uninstall --help
->   ```
+> Examples of the old syntax:
 >
 > - List the installed templates and details about them, including how to uninstall them:
 >
 >   ```dotnetcli
->   dotnet new uninstall
+>   dotnet new --uninstall
 >   ```
 >
 > - Uninstall the Azure web jobs project template package:
 >
 >   ```dotnetcli
->   dotnet new uninstall Microsoft.Azure.WebJobs.ProjectTemplates
+>   dotnet new --uninstall Microsoft.Azure.WebJobs.ProjectTemplates
 >   ```
 
+## Arguments
+
+- **`<PATH|NUGET_ID>`**
+
+  The folder on the file system or the NuGet package identifier the package was installed from. Note that version for NuGet package should not be specified.
+
+## Options
+
+- **`-h|--help`**
+
+  Prints out help for the uninstall command. Available since .NET SDK 7.0.100.
+
+- **`-d|--diagnostics`**
+
+  Enables diagnostic output. Available since .NET SDK 7.0.100.
+
```

Comment:
```suggestion
- **`-d|--diagnostics`**

  Enables diagnostic output. Available since .NET SDK 7.0.100.

- **`-h|--help`**

  Prints out help for the uninstall command. Available since .NET SDK 7.0.100.

```